src/apps/content-editor/src/app/components/Editor/Field/Field.js

```diff
@@ -453,23 +467,30 @@ export default connect(state => {
         }
       }
 
-      const onOneToOneOpen = useCallback(
-        () => Promise.resolve(dispatch(fetchItems(relatedModelZUID))),
-        []
-      );
+      const onOneToOneOpen = useCallback(() => {
+        return dispatch(
+          fetchItems(relatedModelZUID, {
+            lang: props.allLanguages.find(lang => lang.ID === langID).code
```

Comment:
This function happens twice. Maybe worth hoisting out of the switch statement to a `selectedLang` variable?